db/migrate/20180926091756_add_default_value_to_feedback_metrics.rb

```diff
@@ -0,0 +1,21 @@
+class AddDefaultValueToFeedbackMetrics < ActiveRecord::Migration[5.2]
+  def up
```

Comment:
I like this migration. Thanks for sorting this out. I think it might be sensible to add a Data Model, as per https://medium.com/@aweirdcraft/backfilling-data-in-rails-5-0-few-thoughts-on-writing-safe-migrations-1f4c8c4454c9